new/types/manager-types.yaml

```diff
@@ -561,6 +617,18 @@ node_types:
         description: AMQP to InfluxDB Broker Source URL
         type: string
         default: { get_input: amqpinflux_source_url }
+      rabbitmq_username:
+        description: RabbitMQ username for access to queues.
+        type: string
+        default: { get_input: rabbitmq_username }
+      rabbitmq_password:
+        description: RabbitMQ password for access to queues.
+        type: string
+        default: { get_input: rabbitmq_password }
+      rabbitmq_ca_cert:
```

Comment:
I see we have different names for this property in different node types. maybe they should be the same? easier to find and handle...